web-common/src/features/chat/core/input/ChatInput.svelte

```diff
@@ -71,6 +70,12 @@
     editor.commands.startMention();
   }
 
+  function startChat(prompt: string) {
+    editor.commands.setContent(prompt);
+    draftMessageStore.set(prompt);
```

Comment:
It looks like this `draftMessageStore.set` command is redundant to `editor.commands.setContent`. Is it necessary? Can you add a comment if so?

Reply:
Actually it is not needed since I added `tick`. Removing it